src/filtrering/filtrering-mine-filter/mine-filter-rad.tsx

```diff
@@ -1,45 +1,46 @@
-import {MineFilter} from '../../ducks/mine-filter';
 import {useDispatch, useSelector} from 'react-redux';
 import {AppState} from '../../reducer';
 import {finnSideNavn, mapVeilederIdentTilNonsens} from '../../middleware/metrics-middleware';
 import {logEvent} from '../../utils/frontend-logger';
 import {velgLagretFilter} from '../../ducks/filtrering';
-import {apneMineFilterModal} from '../../ducks/mine-filter-ui';
+import {apneMineFilterModal, markerMineFilter} from '../../ducks/lagret-filter-ui-state';
 import {Radio} from 'nav-frontend-skjema';
 import RedigerKnapp from '../../components/knapper/rediger-knapp';
 import React, {RefObject, useRef} from 'react';
 import {antallFilter} from '../../components/modal/mine-filter/mine-filter-utils';
 import {ListevisningType} from '../../ducks/ui/listevisning';
+import {LagretFilter} from "../../ducks/lagretFilter";
 
 interface LagretFilterRadProps {
```

Comment:
this is probably MineFilterRadProps, and the function should be MineFilterRad instead of LagretFilterRad

Reply:
Yes, idk how I renamed this one.. 🤦